modules/react/tabs/lib/TabsList.tsx

```diff
@@ -27,7 +28,7 @@ export interface TabListProps extends Omit<Partial<ExtractProps<typeof Stack, ne
    *   {(item) => <Tabs.Item>{item.text}</Tabs.Item>}
    * </Tabs.List>
    */
-  children: ((item: any) => React.ReactNode) | React.ReactNode;
+  children: ((item: T) => React.ReactNode) | React.ReactNode;
```

Comment:
nice this is looking much better!

Reply:
`any` still be as a default of `T` here instead of `unknown`, because `unknown` causes issue with incompatibility of types of provided children function and this children in components based on `Menu`. This issue start displayed because of some changes made in `@types/react` v17, but as we didn't update it with react 17 it shows now.